src/main/java/edu/greenblitz/robotName/utils/shootingCalculations/ShootingZone.java

```diff
@@ -0,0 +1,74 @@
+package edu.greenblitz.robotName.utils.shootingCalculations;
+
+
+import edu.greenblitz.robotName.shootingStateService.ShootingStateCalculations;
+import edu.greenblitz.robotName.utils.GBCircle;
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShootingZone extends GBCircle {
+
+    private List<Pair<Translation2d, Translation2d>> restrictedBounds; //first is the lower bound, second is the upper
```

Comment:
Use a wrapper class instead of a comment
(In some languages there is a method to give a nickname in one line instead of creating an entire class, I'm not sure how to do in Java, look online)